<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditDialog</class>
 <widget class="QDialog" name="EditDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>230</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit Medication</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>183</y>
     <width>304</width>
     <height>30</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="okButton">
      <property name="text">
       <string>OK</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="cancelButton">
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLineEdit" name="concLineEdit">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>150</y>
     <width>329</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="nameLineEdit">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>50</y>
     <width>329</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="errorLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>214</y>
     <width>329</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>78</y>
     <width>329</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Dose (mg)</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="doseLineEdit">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>100</y>
     <width>329</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>28</y>
     <width>329</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Medication Name:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>128</y>
     <width>329</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Concentration (mg/ml)</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
